docs/contents/components/forms/select/index.ja.mdx

```diff
@@ -73,6 +73,32 @@ return <Select placeholder="キャラクターを選択" items={items} />
 </Select>
 ```
 
+### ヘッダーを追加する
+
+```tsx
+<Select
+  placeholder="キャラクターを選択"
+  header={<header>ここにヘッダーが追加されます</header>}
+>
+  <Option value="孫悟空">孫悟空</Option>
+  <Option value="ベジータ">ベジータ</Option>
+  <Option value="フリーザ">フリーザ</Option>
+</Select>
+```
+
+### フッターを追加する
+
+```tsx
+<Select
+  placeholder="キャラクターを選択"
+  footer={<footer>ここにフッターが追加されます</footer>}
```

Comment:
Change

```jsx
footer={<footer>ここにフッターが追加されます</footer>}
```

to

```
footer={<Center pb="2" px="3">ここにフッターが追加されます</Center>}
```

Reply:
[d17fba8](https://github.com/yamada-ui/yamada-ui/pull/2634/commits/d17fba86374ba91f3cd89c73516eea6942b57e3b)